README.md

```diff
@@ -1,24 +1,73 @@
-# README
+# Bcycom Back End
+> This is the backend for the [Bcycom](http://sleepy-forest-22507.herokuapp.com/) web application and below is what you can following to set up this web application...
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+### Deployment
 
-Things you may want to cover:
+[Base url](http://sleepy-forest-22507.herokuapp.com/)
 
-* Ruby version
+## Bcycom Front End
 
-* System dependencies
+[Front End Repo](https://github.com/Dagic-zewdu/final-capstone-frontend)
 
-* Configuration
+## Built With
 
-* Database creation
+- Rails
+- Ruby 
+- Postgresql
+- JWT
+- Heroku
 
-* Database initialization
+## Getting Started
 
-* How to run the test suite
+Here are the steps to follow in order to get this project on your local computer.
 
-* Services (job queues, cache servers, search engines, etc.)
+### Prerequisites
 
-* Deployment instructions
+`rails, '~> 7.0.3' +`
 
-* ...
+`ruby '3.1.2' +`
+
+### Setup
+
+clone this repo by typing `git clone git@github.com:oluyaratosin123/Final-Capstone-Back-End.git`
+
+### Install
+
+install the dependencies by typing `bundle install`
+
+### Usage
+
+start the local server by running `rails s`
+
+
+## Author
+
```

Comment:
You really ought to have API documentation, what do endpoints do, etc? You can also simply use the ruby gem for swagger.